nix/pre-commit.nix

```diff
@@ -0,0 +1,14 @@
+{ ... }:
+{
+  perSystem = { pkgs, lib, ... }: {
+    pre-commit = {
+      check.enable = true;
+      settings = {
+        hooks = {
+            eclint.files = "\.sql$";
```

Comment:
In line with @knutties 's comment above, is there still a reason to limit eclint (EditoConfig linter) to sql files?